crates/swc_ecma_minifier/tests/fixture/next/41992/1/output.js

```diff
@@ -0,0 +1,184 @@
+                            });
+                        },
+                        className: "body-param-content-type",
+                        ariaLabel: "Request content type"
+                    }))), m.default.createElement("div", {
+                        className: "opblock-description-wrapper"
+                    }, m.default.createElement(x, {
+                        setRetainRequestBodyValueFlag: (e)=>v.setRetainRequestBodyValueFlag({
+                                value: e,
+                                pathMethod: g
+                            }),
+                        userHasEditedBody: E.hasUserEditedBody(...g),
+                        specPath: (0, c.default)(l).call(l, 0, -1).push("requestBody"),
+                        requestBody: R,
+                        requestBodyValue: E.requestBodyValue(...g),
+                        requestBodyInclusionSetting: E.requestBodyInclusionSetting(...g),
+                        requestBodyErrors: E.requestBodyErrors(...g),
+                        isExecute: A,
+                        getConfigs: i,
+                        activeExamplesKey: E.activeExamplesMember(...g, "requestBody", "requestBody"),
+                        updateActiveExamplesKey: (e)=>{
+                            this.props.oas3Actions.setActiveExamplesMember({
+                                name: e,
+                                pathMethod: this.props.pathMethod,
+                                contextType: "requestBody",
+                                contextName: "requestBody"
+                            });
+                        },
+                        onChange: (e, t)=>{
+                            if (t) {
+                                const r = E.requestBodyValue(...g), a = y.Map.isMap(r) ? r : (0, y.Map)();
+                                return v.setRequestBodyValue({
+                                    pathMethod: g,
+                                    value: a.setIn(t, e)
+                                });
+                            }
+                            v.setRequestBodyValue({
+                                value: e,
+                                pathMethod: g
+                            });
+                        },
+                        onChangeIncludeEmpty: (e, t)=>{
+                            v.setRequestBodyInclusion({
+                                pathMethod: g,
+                                value: t,
+                                name: e
+                            });
+                        },
+                        contentType: E.requestContentType(...g)
+                    }))));
+                }
+            }
+            (0, W.default)(ht, "defaultProps", {
+                onTryoutClick: Function.prototype,
+                onCancelClick: Function.prototype,
+                tryItOutEnabled: !1,
+                allowTryItOut: !0,
+                onChangeKey: [],
+                specPath: []
```

Comment:
@jridgewell Do you think it's better to remove this file?

Reply:
Yes. If this file changes in the future, there's very little chance I'll spot the previously buggy code again.

Reply:
Good idea, I'll do it for other patches too